codegen/transforms/grpc_interface.sysl

```diff
@@ -183,7 +183,7 @@ CodeGenTransform:
       ParameterList = ep -> <ParameterList>(:
         ParameterDecl = paramDecl("ctx", "context.Context").ParameterDecl
         ParameterDeclC = ["req"] -> <sequence of ParameterDeclC>(ident:
-          ParameterDecl = paramDecl(ident, "*" + package + "." + methodDef(ep).methodName + "Request").ParameterDecl
+          ParameterDecl = paramDecl(ident, "*" + package + "." + methodDef(ep).methodName + "Request" + ", opts ...restlib.RestRequestOption").ParameterDecl
```

Comment:
This change only affects REST requests, not gRPC requests.

Reply:
This is the downstream service for gRPC which is the interface for REST service, as below example.
interface
```
// GetStuff Client
type GetStuffClient struct {
	PostStuff func(ctx context.Context, req *simple.PostStuffRequest, opts ...restlib.RestRequestOption) (*simple.Str, error)
}
```
handler
```
// GetStuff ...
func (s *GrpcServiceHandler) GetStuff(ctx context.Context, req *pb.GetStuffRequest) (*pb.GetStuffResponse, error) {
	if s.serviceInterface.GetStuff == nil {
		return s.unimpl.GetStuff(ctx, req)
	}

	ctx, cancel := s.genCallback.DownstreamTimeoutContext(ctx)
	defer cancel()
	client := GetStuffClient{
                //here, the simpleSimpleService.PostStuff has the opts
		PostStuff: s.simpleSimpleService.PostStuff,
	}

	return s.serviceInterface.GetStuff(ctx, req, client)
}
```